pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -45,8 +45,17 @@
   flex: 2;
 }
 
-.componentImage {
-  max-width: var(--web-app-component-image-max-width);
+.logoContainer {
+  height: 255px;
+  width: 300px;
+  display: flex;
+  justify-content: center;
+}
```

Comment:
I didn't necessarily mean that we should _completely_ hardcode these values. There is a solution which incorporates both fixed and dynamic values:

```
width: var(--dynamic-width);
height: var(--dynamic-height);
max-width: 300px; // fixed, like it is now
max-height: 255px; // fixed, like it is now
```

Reply:
This way, a theme could choose to make the logos smaller.